### PR TITLE
labwc-actions(5): add missing position option to ShowMenu

### PR DESCRIPTION
**Pull request by johanmalm**
Noticed when merging #2409

### CODE REVIEW

File: docs/labwc-actions.5.scd [hunk 1]
```diff
@@ -124,9 +124,21 @@ Actions are used in menus and keyboard/mouse bindings.
 *<action name="Reconfigure" />*
 	Re-load configuration and theme files.
 
-*<action name="ShowMenu" menu="value" atCursor="yes" />*
+*<action name="ShowMenu"/>*
```
Comment (Consolatis) [inline docs/labwc-actions.5.scd:127]: This makes it seem as if `ShowMenu` as only argument would work. Suggest adding `menu="root-menu"` or something.

Comment (johanmalm) [reply docs/labwc-actions.5.scd]: Fixed.

**Comment by johanmalm** [comment]
Ready to go.

**Comment by Consolatis** [comment]
:)

**Pull request merged**